apiary.apib

```diff
@@ -1052,6 +1058,9 @@ Gives information about various URLs associated with an ad.
 
 #### UTM Tracking Information
 
+**Note: UTM tracking information is only available if AdStage Join has been added to your AdStage plan and a Join task has been completed for the entity.
+Currently, Join available only as a paid Add-On.**
```

Comment:
Should be "Currently, Join is available..."

Reply:
- [ ] typo

Reply:
- [x] fix